daemons/attrd/attrd_commands.c

```diff
@@ -871,7 +874,14 @@ attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
         }
 
     } else {
-        crm_trace("Unchanged %s[%s] from %s is %s", attr, host, peer->uname, value);
+        if (is_force_write && a->timeout_ms && a->timer) {
+ 	    /* Save forced writing and set change flag. */
+ 	    /* The actual attribute is written by Writer after election. */
+ 	    crm_trace("Unchanged %s[%s] from %s is %s(Set the forced write flag)", attr, host, peer->uname, value);
+ 	    a->force_write = TRUE;
+ 	} else {
+            crm_trace("Unchanged %s[%s] from %s is %s", attr, host, peer->uname, value);
+        }
```

Comment:
There appears to be an indentation issue here, maybe tabs vs spaces?

Reply:
Hi Ken,

Sorry...
I fixed the indication.

Many thanks,
Hideo Yamauchi.